src/proposals/20250109/SparkEthereum_20250109.t.sol

```diff
@@ -523,8 +558,16 @@ contract SparkEthereum_20250109Test is SparkTestBase {
         assertEq(rateLimits.getCurrentRateLimit(controller.LIMIT_SUSDE_COOLDOWN()), 0);
         assertEq(susde.convertToAssets(susde.balanceOf(Ethereum.ALM_PROXY)),        0);
 
+        // Confirm proper recharge rate
+        skip(1 hours);
+
+        assertEq(rateLimits.getCurrentRateLimit(controller.LIMIT_USDE_MINT()),      2_083_333.3308e6);
```

Comment:
Can we update all of these to be represented with the expected rate limits, so like
`25_000_000e18 / uint256(1 days) * 1 hours` for example? And then can add any rounding manually

Reply:
done